the-events-calendar.php

```diff
@@ -27,11 +27,10 @@
 Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
+define( 'THE_EVENTS_CALENDAR_DIR', dirname( __FILE__ ) );
+define( 'THE_EVENTS_CALENDAR_FILE', __FILE__ );
```

Comment:
Can I suggest that we stick with PHP Std for these type of Constants? `__TEC_FILE__`